plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -482,115 +482,7 @@ public void testDate()
     }
 
     @Test(dataProvider = "sessionZonesDataProvider")
-    public void testDatetime(ZoneId sessionZone)
```

Comment:
I just moved this test after the other tests (and renamed it), so the tests that insert with MySql come after the tests that insert with Trino.